server/config/config.js

```diff
@@ -0,0 +1,28 @@
+require('dotenv').config();
+
+module.exports = {
+  development: {
```

Comment:
I think this is better if you use dataURL it is much more concise and better than using it like this